.github/workflows/R-CMD-check.yaml

```diff
@@ -58,7 +58,8 @@ jobs:
         with:
           extra-packages:
             any::rcmdcheck,
-            randomForest=?ignore-before-r=4.1.0
+            randomForest=?ignore-before-r=4.1.0,
+            flexsurv=?ignore-before-r=4.0.0
```

Comment:
This stumped me for a moment because flexsurv only says it requires R >=2.15.0, but it seems like its deSolve dependency is really what requires R 4.0.0
https://cran.r-project.org/web/packages/flexsurv/index.html